css/sponsor.css

```diff
@@ -1,13 +1,109 @@
-/*@use 'vars';*/
+/* Text */
+h2 {
+  text-transform: none; }
+
+.heading-pink {
+  font-size: 2rem;
+  color: #f27aaa;
+  font-weight: bold; }
+
+.heading-lavender {
+  font-size: 2rem;
+  color: #b5a1c4;
+  font-weight: bold; }
+
+/* Mobile Text */
+@media only screen and (max-width: 575px) {
+  h2 {
+    font-size: 2rem; }
+  p {
+    font-size: 0.4rem; } }
+
+.demo-section {
+  background-color: #f3f3f3;
+  height: 100%;
+  padding: 1.25rem; }
+
+.demo-container {
+  margin-top: 3rem;
+  padding-left: 15%;
+  padding-right: 15%; }
```

Comment:
let's make the padding a bit less bc the demographic stuff is getting a bit squish, 10% should look better